scss/atoms/typography/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="utf-8">
+		<meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
Please remove this tag: https://github.com/jquery/css-chassis/pull/57#issuecomment-92556189.